flake.nix

```diff
@@ -161,22 +161,22 @@
               };
 
           # The dependencies are in their own scope, so that they don't have to be
-          # in Nixpkgs top level `pkgs` or `nixComponents`.
-          nixDependencies =
+          # in Nixpkgs top level `pkgs` or `nixComponents2`.
+          nixDependencies2 =
```

Comment:
```suggestion
          # The `2` suffix is here because otherwise it interferes with `nixVersions.latest`, which is used in daemon compat tests.
          nixDependencies2 =
```